dashboard/dashboard-session-sso.md

```
@@ -38,7 +38,7 @@ TiDB Dashboardは、SQLベースの[OIDC](https://openid.net/connect/)サイン
     >
     > 入力したパスワードは暗号化されて保存されます。SQLユーザーのパスワードを変更すると、SSOサインインが失敗します。その場合は、パスワードを再入力することでSSOを復旧できます。
 
-6.  **[承認して保存]を**クリックします。
+6.  **[承認して保存]**をクリックします。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)